modules/services/fusuma.nix

```
@@ -0,0 +1,108 @@
+{ config, pkgs, lib, ... }:
+let cfg = config.services.fusuma;
+in with lib; {
```

For consistent style compared to other modules:

```suggestion

with lib; 

let

  cfg = config.services.fusuma;

in {
```